docs/input-jdbc.asciidoc

```diff
@@ -373,6 +374,52 @@ result-set. The limit size is set with `jdbc_page_size`.
 
 Be aware that ordering is not guaranteed between queries.
 
+[id="plugins-{type}s-{plugin}-jdbc_paging_manual_mode"]
+===== `jdbc_paging_manual_mode`
+
+  * Value type is <<boolean,boolean>>
+  * Default value is `false`
+
+Whether to use manual mode during the JDBC paging
+
+If false, your statement will be automatically surrounded by a count query and subsequent multiple paged queries (with `LIMIT` statement, etc.).
+
+If true, multiple queries (without a count query ahead) will be performed with your statement, until no more rows are retrieved.
+You have to write the paging conditions in your statement configuration by self.
```

Comment:
```suggestion
You have to write the paging conditions in your statement configuration by yourself.
```

Reply:
I would like to suggest another slightly different wording: 

"You have to write your own paging conditions in your statement configuration."